docs/other-api/dev-v2.md

```diff
@@ -256,15 +256,13 @@ To use [Mock Service Worker][msw] in development, you'll need to:
 1. Run MSW as part of your app server
 2. Configure MSW to not mock internal "dev ready" messages to the dev server
 
-For example, if you are using [binode][binode] to integrate with MSW,
-make sure that the call to `binode` is within the `remix dev -c` subcommand.
-That way, the MSW server will have access to the `REMIX_DEV_HTTP_ORIGIN` environment variable:
+doing the following will give MSW access to the `REMIX_DEV_HTTP_ORIGIN` environment variable:
```

Comment:
```suggestion
Make sure that you are setting up your mocks for your _app server_ within the `-c` flag so that the `REMIX_DEV_HTTP_ORIGIN` environment variable is available to your mocks:
```